### PR TITLE
Fix panic when workspace is initialized (#120)

### PR DESCRIPTION
**Pull request by dketterer**

### CODE REVIEW

File: bob/init.go [hunk 1]
```diff
@@ -20,9 +22,11 @@ func (b *B) init() (err error) {
 	dir := filepath.Join(b.dir, global.BobWorkspaceFile)
 
 	if file.Exists(dir) {
-		return ErrWorkspaceAlreadyInitialised
+		return usererror.Wrap(ErrWorkspaceAlreadyInitialised)
 	}
 
+	fmt.Println("Initialize Bob workspace...")
```
Comment (puengel) [inline bob/init.go:28]:
```suggestion
	boblog.Log.V(1).Info("Initialize Bob workspace...")
```

Comment (puengel) [reply bob/init.go:28]: Don't forget to add the import ;)

File: bob/init.go [hunk 2]
```diff
@@ -1,8 +1,10 @@
 package bob
 
 import (
+	"fmt"
```
Comment (puengel) [inline bob/init.go:4]:
```suggestion
	"github.com/benchkram/bob/pkg/boblog"
```

**Review by puengel — CHANGES_REQUESTED** [review]

**Review by puengel — APPROVED** [review]

**Comment by dketterer** [comment]
@Equanox can you merge?

**Pull request merged**